### PR TITLE
[plugin.video.uzg] v1.3.2

### PR DESCRIPTION
**Pull request by Opvolger**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.uzg/resources/language/resource.language.en_gb/strings.po [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Kodi Media Center language file
```
Comment (Rechi) [inline plugin.video.uzg/resources/language/resource.language.en_gb/strings.po]: The correct path for Gotham is ` resources/language/English/strings.po`. `resource.language.en_gb` is only for Helix and newer.

Comment (Opvolger) [reply plugin.video.uzg/resources/language/resource.language.en_gb/strings.po]: Sorry, I just wanted to repair subtitles. But also had to implement these changes suddenly.

Comment (Rechi) [reply plugin.video.uzg/resources/language/resource.language.en_gb/strings.po]: Can you remove this file and `resources/language/resource.language.nl_nl/strings.po` as `resources/language/English/strings.po` and `resources/language/Dutch/strings.po` still work for newer Kodi versions.

Comment (Opvolger) [reply plugin.video.uzg/resources/language/resource.language.en_gb/strings.po]: Done.

Comment (Rechi) [reply plugin.video.uzg/resources/language/resource.language.en_gb/strings.po]: Not sure what you have done, but files are still present.
As this PR has been already merged no further action needed, as it doesn't break anything.

Comment (Opvolger) [reply plugin.video.uzg/resources/language/resource.language.en_gb/strings.po]: Sorry, my GUI did not like that files were added and removed. He did not want tore-write history this away with a force commit.
I saw this just too late and quickly adjusted it with command line. But the PR had already happened then.

https://github.com/Opvolger/plugin.video.uzg/tree/Gotham

If there is another adjustment on the Gotham version, I will send the change

**Pull request merged**